telegram/dice.py

```diff
@@ -55,6 +55,8 @@ def de_json(cls, data, bot):
     """:obj:`str`: '🎲'"""
     DARTS = '🎯'
     """:obj:`str`: '🎯'"""
-    ALL_EMOJI = [DICE, DARTS]
-    """List[:obj:`str`]: List of all supported base emoji. Currently :attr:`DICE` and
-    :attr:`DARTS`."""
+    BASKETBALL = '🏀'
+    """:obj:`str`: '🏀'"""
+    ALL_EMOJI = [DICE, DARTS, BASKETBALL]
+    """List[:obj:`str`]: List of all supported base emoji. Currently :attr:`DICE`,
+    :attr:`DARTS` and :attr: `BASKETBALL`."""
```

Comment:
```suggestion
    :attr:`DARTS` and :attr:`BASKETBALL`."""
```
Else the docs won't render correctly. Please see the contribution guide on how to build the docs locally.